sklearn/multiclass.py

```diff
@@ -598,6 +601,10 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
+    target_pairs : ndarray of shape ``(n_classes * (n_classes - 1) // 2, 2)``
```

Comment:
We include a `_` at the end of fitted attributes. Also I am in-favor of using "class" instead of "target", since the rest of the documentation says class.

```suggestion
    class_pairs_ : ndarray of shape ``(n_classes * (n_classes - 1) // 2, 2)``
```